### PR TITLE
Addl ctor for ZuulException to support all args

### PR DESCRIPTION
**Pull request by quidryan**

### CODE REVIEW

File: zuul-core/src/main/java/com/netflix/zuul/exception/ZuulException.java [hunk 1]
```diff
@@ -38,6 +38,17 @@ public ZuulException(String sMessage, Throwable throwable, String errorCause) {
         this.errorCause = errorCause;
     }
 
+    /**
+     * Source Throwable, message, status code and info about the cause
+     * @param sMessage
+     * @param throwable
+     * @param errorCause
+     */
```
Comment (artgon) [inline zuul-core/src/main/java/com/netflix/zuul/exception/ZuulException.java]: missing param :)

Comment (quidryan) [reply zuul-core/src/main/java/com/netflix/zuul/exception/ZuulException.java]: Would it be better if I removed all the @param's? Do they really add necessary documentation?

Comment (artgon) [reply zuul-core/src/main/java/com/netflix/zuul/exception/ZuulException.java]: You can get rid of the params but describe the method better.

**Comment by carl-mastrangelo** [comment]
@quidryan Are you still interested in this PR?   I am going through and trying to clean up our PRs and either submit them or close them out.

**Pull request closed without merging**